README.md

```diff
@@ -2,7 +2,7 @@
 [![codecov.io](https://codecov.io/github/fossasia/open-event-android/coverage.svg?branch=master)](https://codecov.io/github/fossasia/open-event-android?branch=master)
 [![todofy badge](https://todofy.org/b/fossasia/open-event-android)](https://todofy.org/r/fossasia/open-event-android)
 [![Join the chat at https://gitter.im/fossasia/open-event-android](https://badges.gitter.im/fossasia/open-event-android.svg)](https://gitter.im/fossasia/open-event-android?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-
+[![Android Arsenal](https://img.shields.io/badge/Android%20Arsenal-Appetize.io-blue.svg?style=flat)](https://appetize.io/app/2rfx5pavny47jnb1qzwg204fr8)
```

Comment:
Just make a hyperlink or use appetize's logo.

AA just makes 0 sense
On 08-May-2016 23:24, "Harshit Dwivedi" <notifications@github.com> wrote:

> In README.md
> <https://github.com/fossasia/open-event-android/pull/359#discussion_r62435829>
> :
>
> > @@ -2,7 +2,7 @@
> >  [![codecov.io](https://codecov.io/github/fossasia/open-event-android/coverage.svg?branch=master)](https://codecov.io/github/fossasia/open-event-android?branch=master)
> >  [![todofy badge](https://todofy.org/b/fossasia/open-event-android)](https://todofy.org/r/fossasia/open-event-android)
> >  [![Join the chat at https://gitter.im/fossasia/open-event-android](https://badges.gitter.im/fossasia/open-event-android.svg)](https://gitter.im/fossasia/open-event-android?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
> > -
> > +[![Android Arsenal](https://img.shields.io/badge/Android%20Arsenal-Appetize.io-blue.svg?style=flat)](https://appetize.io/app/2rfx5pavny47jnb1qzwg204fr8)
>
> @championswimmer <https://github.com/championswimmer> I generated the
> badge from AA.
> Couldn't find any other badge for appetite.io
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/fossasia/open-event-android/pull/359/files/1290f94ae394addf24a9e0c6b97518ca8d35f50b#r62435829>
>